discord_bot_aternos.py

```diff
@@ -1,81 +1,95 @@
-load_dotenv()
+import interactions
+
+#In the Discord developer portal remember to surely enable messages in privileged intents else it won't work
+
+bot = interactions.Client(os.getenv("TOKEN"))
 
-# Set up Discord client with message PRIVILEGED intents enabled in DEVELOPER PORTAL
-intents = discord.Intents(messages=True)
-intents.message_content = True
-client = discord.Client(intents=intents)
 
 
 help_message = "```Available commands:\n" \
-               "!startserver - Start the Aternos server.\n" \
-               "!serverstatus - Check the status of the Aternos server.\n" \
-               "!playercount - Check the number of players on the Aternos server.\n" \
-               "!playerlist - Check the list of players on the Aternos server.\n" \
-               "!serveraddress - Check the address of the Aternos server.```"
+               "/startserver - Start the Aternos server.\n" \
+               "/serverstatus - Check the status of the Aternos server.\n" \
+               "/playercount - Check the number of players on the Aternos server.\n" \
+               "/playerlist - Check the list of players on the Aternos server.\n" \
+               "/serveraddress - Check the address of the Aternos server.```"
+
 
-@client.event
+
+@bot.event
 async def on_ready():
     print('Bot is ready.')
 
 
-@client.event
-async def on_message(message):
-
-    # Fill in your ATERNOS CREDENTIALS in env file
+@bot.command(name="startserver",description="Starts Minecraft Server")
+async def startserver(ctx: interactions.CommandContext):
     api = Client.from_credentials(os.getenv('ATERNOS_USERNAME'), os.getenv('ATERNOS_PASSWORD'))
+    servers = api.list_servers()
+    server = servers[2]
+
+    if server.status_num == Status.on:
+        await ctx.send('Server is already **online**!')
+    elif server.status_num == Status.off:
+        await ctx.send('Starting Aternos server. Please wait patiently...')
+        server.start()
+    
 
-    # Get list of Aternos servers and select the first one
+@bot.command(name="serverstatus",description="Status of Minecraft Server")
+async def serverstatus(ctx: interactions.CommandContext):
+    api = Client.from_credentials(os.getenv('ATERNOS_USERNAME'), os.getenv('ATERNOS_PASSWORD'))
+    servers = api.list_servers()
+    server = servers[2]
+    
+    status_msg = {
+            Status.on: 'Server is online!',
+            Status.off: 'Server is offline!',
+            Status.starting: 'Server is starting!',
+            Status.loading:'Server is Loading',
+            Status.preparing:'Server is Preparing'
+        }
+    await ctx.send(f'```{status_msg[server.status_num]}```')
+
+
+@bot.command(name="playercount",description="Number of players playing in Minecraft Server")
+async def playercount(ctx: interactions.CommandContext):
+    api = Client.from_credentials(os.getenv('ATERNOS_USERNAME'), os.getenv('ATERNOS_PASSWORD'))
     servers = api.list_servers()
-    server = servers[1]
+    server = servers[2]

```

Comment:
looks like repeating same logic in all bot commands. call internal function to get server